server/src/main/java/org/elasticsearch/action/admin/indices/alias/get/GetAliasesResponse.java

```diff
@@ -51,23 +50,16 @@
 
     private ImmutableOpenMap<String, List<AliasMetaData>> aliases = ImmutableOpenMap.of();
     private RestStatus status = RestStatus.OK;
-    private String errorMsg = "";
+    private String errorMessage;
 
-    public GetAliasesResponse(ImmutableOpenMap<String, List<AliasMetaData>> aliases, RestStatus status, String errorMsg) {
+    public GetAliasesResponse(ImmutableOpenMap<String, List<AliasMetaData>> aliases, RestStatus status, String errorMessage) {
         this.aliases = aliases;
-        if (status == null) {
-            this.status = RestStatus.OK;
-        }
-        this.status = status;
-        if (errorMsg == null) {
-            this.errorMsg = "";
-        } else {
-            this.errorMsg = errorMsg;
-        }
+        this.status = status == null ? RestStatus.OK : status;
+        this.errorMessage = errorMessage;
     }
 
     public GetAliasesResponse(ImmutableOpenMap<String, List<AliasMetaData>> aliases) {
```

Comment:
would it be possible to remove this constructor? It seems that it's only used in tests and we could rather use the above constructor and provide null arguments to it.